frontend/app/routes/public/apply/$id/terms-and-conditions.tsx

```diff
@@ -33,6 +34,19 @@ export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
 export async function loader({ context: { appContainer, session }, request, params }: LoaderFunctionArgs) {
   loadApplyState({ params, session });
 
+  const applicationYears = await appContainer.get(TYPES.domain.services.ApplicationYearService).listApplicationYears({ date: '2020-01-01', userId: 'test' });
+
+  const applicationYearsSchema = z
```

Comment:
I'm not sure why you have to parse DTOs coming from `listApplicationYears` since it should already be mapped correctly.